.gitignore

```diff
@@ -6,6 +6,7 @@ dist/*
 node_modules/
 *.log
 sample_tests/
+config.js
```

Comment:
Your config never got ignored.... looks like you're going to need to generate a new key :(

Also, would suggest using [dotenv](https://www.npmjs.com/package/dotenv) instead of a config file. If you're down to use dotenv, just replace this line in the gitignore with `.env` and then create a `example.env` with a dummy token value